hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/AbstractCSQueue.java

```diff
@@ -1189,84 +1190,72 @@ boolean canAssignToThisQueue(Resource clusterResource,
 
   }
 
+  private static String ensurePartition(String partition) {
+    return Optional.ofNullable(partition).orElse(RMNodeLabelsManager.NO_LABEL);
+  }
+
+  @FunctionalInterface
+  interface Counter {
+    void count(String partition, Resource reservedRes);
```

Comment:
Nit: The parameter for the Resource is called reservedRes, however when you are calling this method from various places, the resource is not always a reserved resource. I do suggest to rename to "res" or "resource", that is a more generic name for this.

Reply:
Renamed to resource.